test/integration/targets/freebsd_update/tasks/tests.yml

```diff
@@ -0,0 +1,28 @@
+- name: update a newly installed FreeBSD system
+  freebsd_update:
+      action: fetch_install
+      flags: '--not-running-from-cron'
```

Comment:
Modules that accept arbitrary arguments to pass to the underlying commands used are heavily discouraged.

It seems like this flag is something that the module itself should be setting, rather than expecting the consumer of the module to set it.

Reply:
You're right, fixed. Thanks!